recipe/meta.yaml

```diff
@@ -39,25 +39,21 @@ outputs:
       build:
         - {{ compiler('c') }}
         - {{ compiler('cxx') }}
-        - armpl  # [aarch64]        
+        - armpl  # [aarch64]
         - ninja-base
         - pkg-config
       host:
         - python
         - pip
         - cython >=0.29.34,<3.1
+        - meson-python >=0.15.0,<0.16.0
         - python-build
-        # dependencies of vendored meson-python
-        - pyproject-metadata >=0.7.1
-        - tomli >=1.0.0         # [py<311]
-        - setuptools >=60.0     # [py>=312]
-        - colorama              # [win]
         # blas
         - mkl-devel  {{ mkl }}  # [blas_impl == "mkl"]
         - openblas-devel {{ openblas }}  # [blas_impl == "openblas"]
       run:
         - python
-  
+
   # When building out the initial package set for a new Python version / MKL version the
```

Comment:
Have you already built numpy-base? Or even though the PR says, `numpy`, are you intending to build `numpy-base` first? I'm not really sure how the building a multi-output recipe for a new python version is supposed to work, so I'm just going off this comment.

Reply:
Great question. There is no specific action to take here, both outputs are built by this PR. The variable is used when updating working on an initial build for a new python version (which has already been done). The variable is used to break the cyclic dependency between mkl_fft, mkl_random and numpy. (mkl_random depends on numpy-base, and numpy depends on mkl_random and numpy-base)